src/lib/registration/ui/migration_selection_dialog.rb

```diff
@@ -134,14 +144,51 @@ def update_details
       # @param [Integer] migration index
       # @return [String] user friendly description (in RichText format)
       def migration_details(idx)
-        # TODO: display some more user friendly details
-        details = migrations[idx].map do |product|
-          "<li>" + CGI.escapeHTML("#{product.identifier}-#{product.version}-#{product.arch}") +
-            "</li>"
+        details = sorted_migrations[idx].map do |product|
+          installed = installed_products.find do |installed_product|
+            installed_product["name"] == product.identifier
+          end
+
+          "<li>" + product_summary(product, installed) + "</li>"
         end
 
         details = "<ul>" + details.join + "</ul>"
-        _("<h3>Migration Products Details</h3>%s") % details
```

Comment:
Hm, I would split this, because the translation team should not take care about the html tags.